templates/zerver/help/send-group-pm.md

```diff
@@ -0,0 +1,72 @@
+# Group Private Messages
+Users can send group private messages in Zulip.
+## There are three ways to send a Group Private Message:
+### One:
+To send a new group private message,
+first click on the `New private message`
+button located at the bottom of your screen.
+
+![New private message](/static/images/help/private-message.png)
+
+The following box will be displayed:
+
+![New stream message](/static/images/help/private-box.png)
+
+In the **one or more people...** field,
+input the usernames or email addresses of the users
+you want to send a group private message to.
+When you begin typing an email address or username,
+autocomplete suggestions will appear for individuals.
+The list of users in this field creates the group
+to which the private message will be sent.
+
+If you attempt to input a nonexistent username or email address,
+an error message will appear.
+
+### Two:
+If you have already private messaged a group of people,
+their names will appear under the **Group PMs** heading
+at the bottom of the user list on the right side of your screen.
+
+![Group PMs](/static/images/help/group-pms.png)
+
+Clicking on one of the groups will bring up a messaging box.
+The **one or more people...** field will automatically be filled
+with the users in that existing group.
+
+![pm box](/static/images/help/pm-box-auto-filled-field.png)
+
+You can then type your message to that group of users
+in the **Compose your message here...** field.
+
+### Three:
+Within the **Private messages** tab on the left of the screen, you can view and respond to group messages.
+
+![Private messages sidebar](/static/images/help/pms_in_left_sidebar.png)
+
+When you click on the **Private messages** tab all of your private
+messages will be shown in the center of your screen.
+
+![Private messages](/static/images/help/private-messages.png)
+
+Clicking on a message from a group private message in this view
+will bring up a messaging box with the **one or more people...** field
+filled with the users in that group private message.
+You can then type a message to that group in the
+**Compose your message here...** field.
+
+Also, you can see your group private messages under **Private messages**
+at the top left corner of your screen.
+
+![Groups left sidebar](/static/images/help/groups-left-sidebar.png)
+
+Clicking on one of these groups will display only messages from that
```

Comment:
May be use 'narrow to that group' as it is the name of that feature. We can also link to the narrowing docs.